<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>161</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="fromUserbox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>From</string>
   </property>
  </widget>
  <widget class="QComboBox" name="toUserbox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>To</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="manage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>76</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Manage</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="sendonbehalf">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Send on behalf</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="sendas">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>76</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Send As</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="forwarding">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Forwarding</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submitBtn2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="remove">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>76</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>remove</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
